src/views/ArtistList.jsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from "react-router-dom";
+
+const artist = [
+  { 
+    id: 1,
+    name: 'Taylor Swift' 
+  },
+  { 
+    id: 2,
+    name: 'Kanye West' 
+  },
+];
+
+const ArtistList = () => {
+  return(
+    <section class= 'container'>
+      <Link to='/'>Home</Link>
+      <h2>Artists</h2>
+      {artist.map((artist) => <div key={artist.id}><Link to={`/artist/${artist.id}`}>{`${artist.name}`}</Link></div>)}
```

Comment:
Esto quedaría un poco más ordenado en varias líneas (sospecho que sería algo que ESLint atajaría, pero queda pendiente para otro PR configurarlo):
```suggestion
      {artist.map((artist) => (
        <div key={artist.id}>
          <Link to={`/artists/${artist.id}`}>{`${artist.name}`}</Link>
        </div>
      ))}
```